text/0308-cli-advisories.md

```diff
@@ -0,0 +1,280 @@
+
+
+17061   Error when building EKS cluster with monocdk import
+
+        Overview: When using monocdk/aws-eks to build a stack containing
+                  an EKS cluster, error is thrown about missing 
+                  lambda-layer-node-proxy-agent/layer/package.json.
+         
+        Affected versions: >=1.126.0 <=1.130.0.
+
+        More information at: https://github.com/aws/aws-cdk/issues/17061 
+
+If you don’t want to see an advisory anymore, use "cdk acknowledge ID". For example, "cdk acknowledge 16603".
+```
+
+By acknowledging a particular advisory, it won’t show anymore in subsequent calls:
+
+```
+$ cdk acknowledge 16603
+
+ADVISORIES
+
+17061   Error when building EKS cluster with monocdk import
+
+        Overview: When using monocdk/aws-eks to build a stack containing
+                  an EKS cluster, error is thrown about missing 
+                  lambda-layer-node-proxy-agent/layer/package.json.
+         
+        Affected versions: >=1.126.0 <=1.130.0.
+
+        More information at: https://github.com/aws/aws-cdk/issues/17061 
+
+If you don’t want to see an advisory anymore, use "cdk acknowledge ID". For example, "cdk acknowledge 17061".
+```
+
+You can suppress all warnings per individual execution:
+
+```
+$ cdk deploy --no-advisories
+```
+
+And you can disable all advisories indefinitely by adding this entry to
+`~/.cdk.json`:
+
+```
+"advisories": false
+```
+
+Regardless of the state of this flag and the advisories you have acknowledged,
+you can always show the currently active advisories:
+
+```
+$ cdk advisories
+```
+
+This command returns zero if there is no advisory and non-zero otherwise. Users
+can then plug this into a pipeline approval workflow and expect manual review if
+there are any advisories.
+
+> Please note that the acknowledgements are made project by project. If you
+acknowledge an advisory in one CDK project, it will still appear on other 
+projects when you run any CDK commands, unless you have suppressed or disabled 
+advisories.  
+
+### Runbook section (internal to the CDK team)
+
+In case of a high-impact issue, follow these steps:
+
+1. Create or update an issue for this incident on the aws-cdk GitHub repository.
+2. Update the file `advisories.json` on repository `cdklabs/aws-cdk-advisories`, adding an entry for
+   the incident. Example:
+```json
+  {
+    "title":  "Toggling off auto_delete_objects for Bucket empties the bucket",
+    "issueUrl": "https://github.com/aws/aws-cdk/issues/16603",
+    "overview": "If a stack is deployed with an S3 bucket with auto_delete_objects=True, and then re-deployed with auto_delete_objects=False, all the objects in the bucket will be deleted.",
+    "component": ["framework"],
+    "version": "<1.126.0"
+  }
+```
+
+3. Create a PR with this change and wait for an approval. Only SDMs have
+   permission to approve PRs in this repository.
+4. When the PR gets merged, the advisory will be visible to all CLI
```

Comment:
Add an item that says that a PR check will validate `advisories.json` for schema or logical issues, and maybe list a bunch of things we want to check. For example:

1. Schema of the file.
2. Check that the issue exists
3. Length limitation of the title & overview
4. Capitalization?
5. Valid semantic versions
6. Valid components